src/library/scala/collection/mutable/ArrayBuffer.scala

```diff
@@ -66,7 +66,13 @@ class ArrayBuffer[A] private (initialElements: Array[AnyRef], initialSize: Int)
 
   /** Ensure that the internal array has at least `n` cells. */
   protected def ensureSize(n: Int): Unit = {
-    array = ArrayBuffer.ensureSize(array, size0, n)
+    array = ArrayBuffer.ensureSize(array, size0, n.toLong)
```

Comment:
`n.toLong` is not really necessary here, is it? In other places the conversion is done before adding, to make it does not overflow. Here is is just passed. Making the conversion explicit does no harm, though, and it makes it obvious what is going on.

Reply:
I mean, the method takes a `Long`, so I called `toLong`. I guess I forgot that the compiler would insert the conversion for me